flink-runtime/src/main/java/org/apache/flink/runtime/client/JobAttachmentClientActor.java

```diff
@@ -114,7 +114,7 @@ else if (JobClientMessages.getRegistrationTimeout().equals(message)) {
 					client.tell(
 						decorateMessage(new Status.Failure(
 							new JobClientActorRegistrationTimeoutException("Registration for Job at the JobManager " +
-								"timed out. " +	"You may increase '" + ConfigConstants.AKKA_CLIENT_TIMEOUT +
+								"timed out. " +	"You may increase '" + AkkaOptions.AKKA_CLIENT_TIMEOUT +
```

Comment:
`ConfigConstants.AKKA_CLIENT_TIMEOUT` is only the key, so we should only contain the key of the ConfigOption, i.e `AkkaOptions.AKKA_CLIENT_TIMEOUT.key()`.